specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/examples/Pricings/ListPricings_example.json

```diff
@@ -1,46 +1,76 @@
 {
-  "parameters": {
-    "api-version": "2018-06-01",
-    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23"
-  },
-  "responses": {
```

Comment:
why are those parameters and body removed from this example?

Reply:
Because they've been replaced with the correct output.

Reply:
This example is not a valid one. You still need to provide the parameters and replace the response body only.
You could reference this one as an example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/examples/ApiManagementCreateApiIssue.json